_docs_integrate/delete-attributes.md

```diff
@@ -69,8 +69,10 @@ Before sending the Request, we recommend to [validate its content]({% link _docs
 Next, send the Request to the peer.
 You can either do so by [Message]({% link _docs_integrate/data-model-overview.md %}#message) or by a [RelationshipTemplate]({% link _docs_integrate/data-model-overview.md %}#relationshiptemplate), using the `onExistingRelationship` property of a [RelationshipTemplateContent]({% link _docs_integrate/data-model-overview.md %}#relationshiptemplatecontent).
 For a detailed explanation check out our guides on how to send [Requests via Messages]({% link _docs_integrate/requests-via-messages.md %}) and [Requests via RelationshipTemplates]({% link _docs_integrate/requests-via-relationshiptemplates.md %}).
+Once the Request is sent, the according own shared Attribute of the Sender gets a [LocalAttributeDeletionInfo]({% link _docs_integrate/data-model-overview.md %}#localattributedeletioninfo).
```

Comment:
Even though the introductory part of the sentence is "Once the Request is sent", I'd write "Sender of the Request" instead of "Sender", as the terminology "Sender/Recipient" is not heavily used in this guide and this is the first time in this guide where the term "Sender" occurs.

Reply:
I'd prefer to keep the sentence simpler, since as you point out, it should be clear what "Sender" is referring to.